crowdsourcing/viewsets/project.py

```diff
@@ -105,7 +105,14 @@ def destroy(self, request, *args, **kwargs):
 class ModuleViewSet(viewsets.ModelViewSet):
     queryset = Module.objects.all()
     serializer_class = ModuleSerializer
-    #permission_classes=[IsOwnerOrReadOnly, IsAuthenticated]
+    permission_classes=[IsOwnerOrReadOnly, IsAuthenticated]
+
+    @list_route(methods=['get'])
+    def get_prototype_task(self, request, **kwargs):
+        prototype_task = Module.objects.all().filter(project=self.request.query_params.get('projectId'), \
+                                                        name="Prototype Task")[0]
```

Comment:
we can take the first() milestone here instead of relying on the name, it's safe